lib/util.js

```diff
@@ -423,3 +423,21 @@ function compareByGeneratedPositionsInflated(mappingA, mappingB) {
   return strcmp(mappingA.name, mappingB.name);
 }
 exports.compareByGeneratedPositionsInflated = compareByGeneratedPositionsInflated;
+
+/**
+ * Strip any JSON XSSI avoidance prefix from the string (as documented
+ * in the source maps specification), and then parse the string as
+ * JSON.
+ */
+function parseJSON(str) {
+  if (str.startsWith(")]}'")) {
+    var index = str.indexOf("\n");
+    if (index < 0) {
+      // No line, return nothing.
+      return null;
+    }
+    str = str.substring(index + 1);
+  }
+  return JSON.parse(str);
+}
+exports.parseJSON = parseJSON;
```

Comment:
nitpicking: I don't like the name as it looks like the only thing it does is parsing JSON :) Maybe "parseSourceMapInput" ?